### PR TITLE
Use node 14

### PR DESCRIPTION
**Pull request by Crash--**
Migration to use Node 14

- Needed to work on sort() since Node 11 brings a new sort algorithm

### CODE REVIEW

**Comment by JF-Cozy** [comment]
j'ai sorti les modifs hors scope => https://github.com/cozy/cozy-ui/pull/1956

Pour Node c'est un sujet de fond en pause

File: react/AppSections/categories.js [hunk 1]
```diff
@@ -48,7 +48,7 @@ export const sorter = (categoryA, categoryB) => {
     (categoryB.value === 'all' && 1) ||
     (categoryA.value === 'others' && 1) ||
     (categoryB.value === 'others' && -1) ||
-    categoryA.label > categoryB.label
+    categoryA.label.localeCompare(categoryB.label)
```
Comment (trollepierre) [inline react/AppSections/categories.js:51]: pointer

Comment (trollepierre) [reply react/AppSections/categories.js:51]: https://github.com/cozy/cozy-ui/pull/2004/files#r777466613

**Comment by trollepierre** [comment]
Upgrade to Node 16 succeeds

**Pull request closed without merging**